packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -55,3 +60,32 @@ export class NodeTracerProvider extends BasicTracerProvider {
     super.register(config);
   }
 }
+
+/**
+ * Two layer merge.
+ * First, for user supplied config of plugin(s) that are loaded by default,
+ * merge the user supplied and default configs of said plugin(s).
+ * Then merge the results with the default plugins.
+ * @returns 2-layer deep merge of default and user supplied plugins.
+ */
+export function mergePlugins(
```

Comment:
Usually I don't prefer to export function only for tests. Perhaps, we can do something like this:

```
export const TEST_ONLY = {
  mergePlugins
};
```
WDYT? Feel free to ignore if you don't agree.

Reply:
I'm not sure what best practice is, but it did feel a little off to export something strictly to test it, will try your above mentioned approach, thanks!

Reply:
Think I would prefer to make it a private member of the class which can be called in tests using the `instance["_privateFunction"]()` syntax

Reply:
til, sgtm

Reply:
done via https://github.com/open-telemetry/opentelemetry-js/pull/980/commits/03d6a482fb5312fad86f13e071891b56b493e61b